### ISSUE TITLE
Start local installed exac_browser

### ISSUE THREAD

**Issue by qpyuan** [konradjk/exac_browser#353]
Thank you for providing the nice introductions for installing exac_browser locally. I have successfully installed it recently with minor modifications and displayed our own data along with ExAC population data. I have a couple of questions here:

1). When I started mongdb ("mongod --dbpath path.to.myDataBasse &" ) and exac.py ("python /home/webdata/exac/LNG_exac_browser/exac.py", seems like I cannot use background run for exac.py), I got many running messages. What is the correct way to run these two command without running messages or to re-direct limited messages?

2). If I want to display the "Allele Count", "Allele Number", "Allele Frequency" just using our own data but the individual variant page along with ExAC populations, can I just replace the AC, AN numbers in "ExAC_HC.0.3.final.vep.vcf.gz" with our own data?

3). I know someone asked before about how to get the coverage with custom data file (bams) but it seems not be answered. Somewhere your team mentioned that coverage was based on 10% samples and use samtools depth to get the coverage.  Can you please provide more detail about this? 

Thank you very much.

qpyuan

**Comment by cankutcubuk** [comment]
@qpyuan did you find any solution for your first question?

**Comment by qpyuan** [comment]
@cankutcubuk Yes, I can re-direct the stderr to a log file (But I am not sure if they are the best way) by starting the database with:
numactl --interleave=all mongod --dbpath $fullPathToDatabaseDir &>> $logDir/$logName
Then run exac.py with:
python $PATH/exac.py > $logDir/$exacLog 2&>1

I also resolved my question #2 by replace the original data with our own AC and AN. I resolved Question  #3 by running  "samtools depth" with the specific exomeSeq regions for the bam files. Use a R script to generate the coverage files.

Currently, I am struggling to deploy this as production so that multiple requests can be served. @konradjk Please let me how I can deploy this as production (I am using centos 7). I am sorry that I know very little of python, flask.

Thanks.

**Comment by cankutcubuk** [comment]
thanks @qpyuan so do you think it is possible to run the database and exac.py in the background of a docker container and deploy the docker in a server?

I will try and let you know if it works or not.

**Comment by cankutcubuk** [comment]
@qpyuan 
I do it like you said but there is no action. Do I miss something?
I run these codes in the folder where the exac.py is located

numactl --interleave=all mongod --dbpath /var/lib/mongodb/  &>> ./mongo.log 
python ./exac.py > ./exacLog.log 2&>1

**Comment by qpyuan** [comment]
@cankutcubuk 
Is the mongodb the directory storing your exac database during "python manage.py load_db"? If yes, please try and check what log file says:
numactl --interleave=all mongod --dbpath /var/lib/mongodb &>> ./mongo.log

That's like what I am using.
